index.html

```diff
@@ -4,63 +4,219 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Figma Template 1</title>
-    <link rel="stylesheet"
-          href="https://fonts.googleapis.com/css2?family=Crimson+Pro">
-    <link rel="stylesheet" href="style.css">   
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Crimson+Pro">
+    <link rel="stylesheet" href="style.css">
+    <title>Leon's Portfolio</title>
 </head>
 <body>
-    <header>
-        <a class="logo">Leon Espitia</a>
-        <nav class="menu_icon">
-            <div>
-              <img src="./assets/icons/Hamburguer.svg" alt=""/>
-            </div>  
-        </nav>
-    </header>
-
     <main>
-        <section class="headline">
-          <h1 class="title">I'm Leon. Happy to see you!</h1>
-          <p class="headline_content">
-            I'm a software developer! I can help you build a product,
-            feature or website Look through some of my work and experience! 
-            If you like what you see and have a project you need coded, 
-            don't hestiate to contact me.
-          </p>
+<!--card_1 starts from here--> 
+        <div>
+            <section class="picture_container">
+                    <img src="./assets/pictures/picture1.jpg" alt=""/>
+                <div class="info_container">   
+                    <h1 class="title">Tonic</h1>
+                    <div class="tonic_1">
+                        <img src="./assets/img-block1/Frame1.png" alt="">   
+                    </div>
+                    <p class="content">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <nav>
+                        <ul>
+                            <li>
+                                <a href="" target="_blank">
+                                    <img src="./assets/img-block1/html.png" alt="">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="" target="_blank">
+                                    <img src="./assets/img-block1/css.png" alt="">
+                                </a>
+                            </li>
+                            <li>
+                                <a href="" target="_blank">
+                                    <img src="./assets/img-block1/javascript.png" alt="">
+                                </a>
+                            </li>
+                        </ul>
+                    </nav>
+                    <div>
+                        <button type="button" class="see_button">See Project</button>
```

Comment:
- [ ] You should stick the Figma design and adjust the Font-weight to 500, border-width to 1px, border-radius 8px, padding 12px, etc.
#### Figma
![image](https://user-images.githubusercontent.com/44057863/170818567-a6523b3f-875c-4fd4-b643-642c171e0356.png)
#### Impl
![image](https://user-images.githubusercontent.com/44057863/170818596-dcf10179-e714-4237-b344-515cfb6354f4.png)